docs/hacking/index.rst

```diff
@@ -8,5 +8,8 @@ This section is meant for people interested in developing on Antidote.
 
    curriculum.rst
    platform.rst
+   selfmedicate/selfmedicate.rst
```

Comment:
Either as part of https://github.com/nre-learning/antidote-selfmedicate/pull/28 or very soon after I'm going to just remove the old selfmedicate and make the vagrant approach the only version. It works well and it's time to make the switch.

So to avoid confusion, let's just keep a single method of running selfmedicate in these docs - the vagrant way. I think the "home page" for this directory could be your `selfmedicate.rst` page which covers the selfmedicate script. There should be an introductory section on this page that covers running the script directly (baremetal) or using the vagrant approach (with an internal link to that page).

Reply:
The changes you suggested has been made.  Please let me know if these changes work.